feat(docs): added missing heatlh, uptime and version docu (#302)
meshcloud/meshcloud-docs: docs/meshstack.monitoring.md

}
```

> ## Health endpoint

Reviewer: It is not a single health endpoint right? We have multiples

Author: Yes. We've the health endpoint (status, db connection status), metrics (uptime) and version.

meshStack configures a health endpoint which returns the current status and also the status of database connections. These information are available under path `/actuator/health`.

Example:

```json
{
    "status": "UP",
    "components": {
        "db": {
            "status": "UP",
            "details": {

>                 "database": "MariaDB",

Reviewer 2: d: Hmm, perhaps it would be ciritical to provide info that we use mariadb via a public endpoint. Can the details be removed?

Author: I know. But I couldn't find any confg to return only the status. Then we need to implement a custom indicator.

Author: I couldn't find a soultion to return only the status of the db connection.
How should we handle this now? @sttomm

Author: @chewbeccy Do you think its ok that we also return the db name? Or is this a security issue?

Reviewer: Not a fan to be honest of sharing DB names publicly. I do see it as a medium risk and would like to mitigate it. But at the moment you dont see any option hiding it?

Author: Currently I expose the db status only with a specific actuator configuration. If we want to hide the db name then we need to implement a custom indicator to expose only the db status. I'm not sure how big the effort is to implement a custom db status indicator.

                "result": 1,
                "validationQuery": "SELECT 1"
            }
        }
    }
}
```

## Uptime

> meshStack configures an uptime endpoint to request the exact process instance uptime. These information is available under path `/actuator/metrics/process.uptime`.

Reviewer 2: ```suggestion
meshStack configures an uptime endpoint to request the exact process instance uptime. This information is available under path `/actuator/metrics/process.uptime`.
```

Author: Fixed

Example:

```json
{
    "name": "process.uptime",
    "description": "The uptime of the Java virtual machine",
    "baseUnit": "seconds",
    "measurements": [
        {
            "statistic": "VALUE",
            "value": 2396.331
        }
    ],
    "availableTags": []
}
```

## Version

> meshStack configures an version endpoint to request the current version and commit-hash. These information are available under path `<meshPanel>/version`.

Reviewer 2: ```suggestion
meshStack configures a version endpoint to request the current version and commit-hash. This information is available under path `<meshPanel>/version`.
```

Author: Fixed
